cli/objects/rule.py

```diff
@@ -274,6 +277,69 @@ def _get_jira_additional_labels(
         else:
             return None
 
+    def _get_jira_assignee(self, rule_dict: dict[Any, Any]) -> Optional[str]:
+        """
+        Determines if a Jira Assignee is defined in a rule. If it is, validate it and return the string.
+
+        :param rule_dict: A dictionary object representing a user-defined firewatch rule.
+        :return: A string of the Jira assignee to use in a firewatch rule. If one is not defined, return None
+        """
+        if "jira_assignee" in rule_dict.keys():
+            try:
+                jira_assignee = rule_dict["jira_assignee"]
+            except Exception as ex:
+                self.logger.error(ex)
+                exit(1)
+            if isinstance(jira_assignee, str):
+                regex = r"\b[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Z|a-z]{2,7}\b"  # Used to check if email is valid
+                if re.fullmatch(regex, jira_assignee):
+                    return jira_assignee
+                else:
+                    self.logger.error(
+                        f'Value for "jira_assignee" is not an email address in firewatch rule: "{rule_dict}"',
+                    )
+                    exit(1)
+            else:
+                self.logger.error(
+                    f'Value for "jira_assignee" is not a string in firewatch rule: "{rule_dict}"',
+                )
+                exit(1)
+        else:
+            return None
+
+    def _get_jira_priority(self, rule_dict: dict[Any, Any]) -> Optional[str]:
+        """
+        Determines if a Jira priority is defined in a rule. If it is, validate it and return the string.
+
+        :param rule_dict: A dictionary object representing a user-defined firewatch rule.
+        :return: A string of the Jira priority to use in a firewatch rule. If one is not defined, return None
+        """
+        valid_priority_values = ["Blocker", "Critical", "Major", "Normal", "Minor"]
+
+        if "jira_priority" in rule_dict.keys():
+            try:
```

Comment:
Similar to the comment here: https://github.com/CSPI-QE/firewatch/pull/23/files#r1275267525

Reply:
Will fix in an upcoming commit!

Reply:
Fixed in [b819732](https://github.com/CSPI-QE/firewatch/pull/23/commits/b819732d735899bd815b3292b116c2abf2d3aa67). Thanks!